src/main/java/org/aion/wallet/ui/components/SettingsController.java

```diff
@@ -57,23 +48,17 @@ protected void registerEventBusConsumer() {
 
     public void changeSettings() {
         final LightAppSettings newSettings;
-        try {
-            newSettings = new LightAppSettings(
-                    address.getText().trim(),
-                    port.getText().trim(),
-                    DEFAULT_PROTOCOL,
-                    settings.getType(),
-                    Integer.parseInt(timeout.getText()),
-                    getSelectedTimeoutMeasurementUnit()
-
-            );
-            displayNotification("", false);
-            EventPublisher.fireApplicationSettingsChanged(newSettings);
-        } catch (ValidationException e) {
-            ConsoleManager.addLog("Could not update settings", ConsoleManager.LogType.SETTINGS, ConsoleManager.LogLevel.WARNING);
```

Comment:
did we lose this here?

Reply:
`ValidationException` was never thrown from this constructor, it was just a redundant `throws` clause and after removing it, there's no need to catch and log any more